Python/bin/packUtil.py

```diff
@@ -1060,8 +1061,10 @@ def packTupleID(triplet):
          tmp = m.group(1)
          #print ("m.groups is ", m.groups())
          if len(tmp) > 7:
-            raise RuntimeError("Can't pack " + trkSub + " because it it is too long")
-         packedTrkSub = m.group(1)
+            print('Warning: trkSub ' + trkSub + ' is too long for obs80, removing the first character')
+            packedTrkSub = tmp[1:]
+         else:
+            packedTrkSub = m.group(1)
```

Comment:
@federicaspoto Wondering if this should be made more robust. What happens here if trkSub is 9 characters? You might want to truncate for 8 characters but for >8 characters just keep the original `raise RunTimeError`. Or just have a more robust truncation approach and pass everything.

Reply:
@stevechesley it won't be possible to have a trksub longer than 8 characters, because the check for trksub will stop it saying that it is an invalid trksub. Also, any file with trksubs with more than 8 characters won't pass the validation. I'll add a test about it.